pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
Ok - will do. Though I suspect this may need some change when the sharding support is added.